pkg/controller/networkpolicyrecommendation/controller.go

```diff
@@ -598,15 +598,19 @@ func (c *NPRecommendationController) startSparkApplication(npReco *crdv1alpha1.N
 	}
 	sparkResourceArgs.executorMemory = npReco.Spec.ExecutorMemory
 
-	recommendationID := uuid.New().String()
+	err = commands.ParseRecommendationName(npReco.Name)
```

Comment:
I understand that using the same uuid for NPR and related SparkApplication may be good in terms of consistency. But maybe move the `ParseRecommendationName` function to `pkg/util`? It is a little bit strange that the controller has dependency on the CLI package.

Reply:
move it to pkg/util/policyrecommendation